src/test/scala/scalachecklib/Test.scala

```diff
@@ -0,0 +1,53 @@
+package scalachecklib
```

Comment:
Let's create an issue to remove this once the scala-exercises core library is upgraded.